Editor/Scenes/SongEditor/SongEditor.gd

```diff
@@ -0,0 +1,59 @@
+extends VBoxContainer
+
+var track_name = preload("./TrackName.tscn")
+
+onready var names = $TracksScroll/HBox/Names
+onready var segments = $TracksScroll/HBox/SegmentScroll/VBoxContainer
+
+# styles
+# techno: Make this support different themes
```

Comment:
Well if i can make a new class in the main theme i can make it in different themes

Reply:
Yeah that can be set for later. Right now it's alright :)